build.gradle

```diff
@@ -942,6 +942,7 @@ project(':core') {
     from(project(':streams').jar) { into("libs/") }
     from(project(':streams').configurations.runtime) { into("libs/") }
     from(project(':streams:streams-scala').jar) { into("libs/") }
+    from(project(':streams:streams-scala').configurations.archives.artifacts.files.filter { file -> file.name.matches(".*scaladoc.*")}) { into("libs/") }
```

Comment:
Do we actually need to do this? I don't see any other line here that would copy JavaDocs-jar into the `lib` directory?

\cc @ijuma